### PR TITLE
Fixed ref to oracle documentation and oracle environment setup documentation

### PR DESCRIPTION
**Pull request by felixxm**
Fixed ref to oracle documentation and oracle environment setup documentation

### CODE REVIEW

File: docs/intro/tutorial02.txt [hunk 1]
```diff
@@ -89,8 +89,8 @@ and creates any necessary database tables according to the database settings
 in your :file:`mysite/settings.py` file and the database migrations shipped
 with the app (we'll cover those later). You'll see a message for each
 migration it applies. If you're interested, run the command-line client for your
-database and type ``\dt`` (PostgreSQL), ``SHOW TABLES;`` (MySQL), or
-``.schema`` (SQLite) to display the tables Django created.
+database and type ``\dt`` (PostgreSQL), ``SHOW TABLES;`` (MySQL), ``.schema`` (SQLite),
```
Comment (timgraham) [inline docs/intro/tutorial02.txt]: please wrap documentation at 79 characters

File: docs/ref/databases.txt [hunk 2]
```diff
@@ -719,7 +719,7 @@ you add quotes where necessary before copying a query into an SQLite shell.
 Oracle notes
 ============
 
-Django supports `Oracle Database Server`_ versions 11.1 and higher. Version
+Django supports `Oracle Database Server`_ versions 11.2 and higher. Version
```
Comment (timgraham) [inline docs/ref/databases.txt]: Have we done something to break support for Oracle 11.1? We documented 11.1 as the minimum version in Django 1.8 -- if we want to drop support for 11.1 we also need to document that in the release notes.

File: docs/intro/tutorial02.txt (hunk 1 above)
Comment (felixxm) [inline docs/intro/tutorial02.txt]: Fixed

File: docs/ref/databases.txt (hunk 2 above)
Comment (jtiai) [inline docs/ref/databases.txt]: Oracle 11.1 is not anymore supported by Oracle - 11.1 EOL was 3 years ago, extended support ended at the end of August 2015.

To download Oracle 11.1 server you actually have to do special request through Oracle Support and download is only available for users that do have a valid Oracle Database License.

Same rationale was used to drop support < 11.1 in 1.8:
https://code.djangoproject.com/ticket/23145

Comment (timgraham) [inline docs/ref/databases.txt]: Thanks for the explanation. Completed the docs for dropping support for 11.1 in https://github.com/django/django/pull/5310.

**Comment by timgraham** [comment]
merged in 6f1b09bb5c1bafe4633514cbff37f9a7ed7a63ae, thanks!

**Pull request closed without merging**